### DISCUSSION TITLE
Is it possible to use this library in ubuntu ?

### DISCUSSION THREAD

**Discussion by vanthuc96** [czproject/git-php#84] [Q&A] [answered]
I'm trying to use this lib, but I got some issues and I guess it depends on the difference of ubuntu and window, so is it possible to use this lib in ubuntu ?

**Comment by janpecha** [comment, marked as answer]
Hi, yes, it is possible to use this library in Ubuntu. I use Ubuntu too.

What issues you have?

**Reply by vanthuc96** [reply]
I try to call the function in GitRepository, but I got this issue: `detected dubious ownership in repository at '/path/myfolder/' To add an exception for this directory, call: git config --global --add safe.directory /path/myfolder`
And I try to add one more function in GitRepository to execute above git command, but I still got this error 'could not lock config file /var/www/.gitconfig: Permission denied'. Do you have any ideas for this? (Thanks for replying my question)

**Reply by janpecha** [reply]
Maybe that repository has another owner than your PHP script?

I found some links for this error message:

* https://stackoverflow.com/questions/72978485/git-submodule-update-failed-with-fatal-detected-dubious-ownership-in-repositor
* https://github.com/git/git/commit/8959555cee7ec045958f9b6dd62e541affb7e7d9

**Reply by vanthuc96** [reply]
Yes, I have tried it, but as I mention above, I got this error `could not lock config file /var/www/.gitconfig: Permission denied`. Do we have any way to set permission for that file in php script ?

**Reply by vanthuc96** [reply]
For this, I just allow user that is running my php script can access to that file in terminal, It's resolved. Thanks for your help @janpecha
